apps/site/src/app/(main)/(home)/sections/FAQ/FAQAccordionMobile.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+
+import React, { useState } from "react";
+import Image from "next/image";
+
+import ListItemButton from "./components/ListItemButton";
+import { FAQAccordion, FAQ } from "./FAQ";
+
+import SpeechMobile from "./assets/speech-mobile.svg";
+
+const FAQAccordionMobile: React.FC<FAQAccordion> = ({ faq }) => {
+	const [focusedQuestion, setFocusedQuestion] = useState<null | FAQ>(null);
+
+	return (
+		<div className="flex flex-1 justify-center lg:h-auto mt-60 lg:mt-8 mb-[-250px] lg:mb-0 sm:text-lg md:text-xl">
+			{/* Changes size of parent component */}
+			<div
+				className={`z-0 ${
+					focusedQuestion ? "h-[520px]" : "h-[965px] sm:h-[940px] md:h-[1030px]"
+				} duration-200 transition-transform lg:hidden`}
+			/>
+			<div className="relative flex justify-center lg:hidden">
+				<Image
+					src={SpeechMobile}
+					alt="Dialogue box background"
+					className={`${
+						focusedQuestion
+							? "mt-[-160px] sm:mt-[-120px] sm:max-h-[620px]"
+							: "mt-[-275px] md:mt-[-300px]"
+					} min-w-[450px] sm:min-w-[600px] md:min-w-[750px] object-fill`}
+					layout="responsive"
+				/>
+
+				{/* No question is focused. */}
+				<div
+					className={`${
+						focusedQuestion
+							? "opacity-0 pointer-events-none"
+							: "opacity-100 z-50"
```

Comment:
Suggestion [repeating]: rather than using `opacity` to show and hide stuff, use `hidden` instead